src/corelibs/U2Core/src/tasks/ExternalToolRunTask.cpp

```diff
@@ -586,6 +601,87 @@ QVariantMap ExternalToolSupportUtils::getScoresGapDependencyMap() {
     return map;
 }
 
+QString ExternalToolSupportUtils::checkArgumentPathSymbols(const QStringList& args) {
```

Comment:
Rename the method to show that it checks for latin1 symbols only.
BTW are sure that all latin1 symbols are OK?